apps/ethvm/src/models/Uncle.ts

```diff
@@ -1,5 +1,5 @@
 import { common } from '@app/helpers'
-import { Hash, Hex, HexNumber, HexTime, Tx } from '@app/models'
+import { Hash, Hex } from '@app/models'
 import { Uncle as UncleLayout } from 'ethvm-common'
```

Comment:
If there's no collision with Uncle just use import { Uncle } from 'ethvm-common' instead of current one

Reply:
thers's an collision